tests/framework/e2e/util.go

```diff
@@ -50,11 +50,11 @@ func SpawnWithExpectWithEnv(args []string, envVars map[string]string, expected s
 }
 
 func SpawnWithExpects(args []string, envVars map[string]string, xs ...string) error {
-	_, err := SpawnWithExpectLines(args, envVars, xs...)
+	_, err := SpawnWithExpectLines(context.TODO(), args, envVars, xs...)
```

Comment:
Have you consider adding one more parameter context to the function `SpawnWithExpects`?

Reply:
A little but stopped as whole expects package needs a cleanup.